Xamarin.Forms.Core/CornerRadius.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace Xamarin.Forms.Core
+{
+	public struct CornerRadius
```

Comment:
require a `[TypeConverter]` so it can be set from XAML and CSS

Reply:
It is true!. I have added the Converter.